src/Evolution/Executables/GeneralizedHarmonic/EvolveGeneralizedHarmonic.hpp

```diff
@@ -281,8 +288,10 @@ struct EvolutionMetavars {
       Initialization::Actions::TimeAndTimeStep<EvolutionMetavars>,
       evolution::dg::Initialization::Domain<volume_dim>,
       Initialization::Actions::NonconservativeSystem,
-      evolution::Initialization::Actions::SetVariables<
-          domain::Tags::Coordinates<volume_dim, Frame::Logical>>,
+      std::conditional_t<
```

Comment:
So I can understand, please explain to me what this addition is doing and why it's necessary?

Reply:
This change makes sure we don't even try to set the initial data from the analytic solution, and therefore prevents initialization code below from using it (which would be a bug). If code before the `InitializeInitialDataDependentQuantities` phase tries to use initial data now we get an FPE (at least in Debug builds).